lib/src/icons.dart

```diff
@@ -58,12 +11,12 @@ enum GYWIcon {
   /// The height (in pixels) of the icon
   final double height;
 
-  const GYWIcon(
-    this.name,
-    this.filename,
-    this.width,
-    this.height,
-  );
+  const GYWIcon({
+    required this.name,
+    required this.filename,
+    required this.width,
+    required this.height,
+  });
 
   /// The path of the associated file in the assets folder
   @Deprecated("Use pathPng instead")
```

Comment:
This property should be removed. Do you have a PR dedicated to deprecated features ? Otherwise, you can remove it here